index.html

```diff
@@ -10,104 +10,123 @@
-          <div class="country-info-switzerland">
-          <img src="images/uil_calender.png"> <p>8 Days</p>
-          <img src="images/location/loca-input-user.png"> <p>25 People going</p>
-        </div>
-        <div class="country-name-switzerland">
-          <h1 class="country-name-text-switzerland">Switzerland</h1>
-          <img class="star-point" src="images/bi_star-fill.png">
-          <img class="star-point" src="images/bi_star-fill.png">
-          <img class="star-point" src="images/bi_star-fill.png">
-          <img class="star-point" src="images/bi_star-fill.png">
-          <img class="star-point" src="images/bi_star-fill.png">
-        </div>
-        <div class="navigation-country-switzerland">
-          <img src="images/ep_location.png">
-          <p class="location-country-text-switzerland">Europe</p>
-        </div>
-        <div class="price-text-switzerland">
-          <h3 class="onemilion">1,000 $</h3>
-          <h6 class="onemiliontwo">1,200 $</h6>
-        </div>
-        <div class="country-description-text-switzerland">
-          <p class="description-text-switzerland">Nam exercitationem commodi et ducimus quia in dolore <br>
-            animi sit mollitia amet id quod eligendi. Et labore harum <br> 
-            non nobis ipsum eum molestias mollitia et corporis <br>
-            praesentium a laudantium internos.</p>
-            <button class="explore-now-btn-switzerland">Exlore Now</button>
-        </div>
-        </div>
-      <div class="brazil-container">
-          <img class="photo-brazil" src="images/hometour-packages-brazil.png">
-          <img class="img-brazil" src="images/brazil-flag.png">
-          <div class="country-info-switzerland">
-            <img src="images/uil_calender.png"> <p>8 Days</p>
-            <img src="images/location/loca-input-user.png"> <p>30 People going</p>
-          </div>
-          <div class="country-name-switzerland">
-            <h1 class="country-name-text-switzerland">Amazon</h1>
-            <img class="star-point" src="images/bi_star-fill.png">
-            <img class="star-point" src="images/bi_star-fill.png">
-            <img class="star-point" src="images/bi_star-fill.png">
-            <img class="star-point" src="images/bi_star-fill.png">
-            <img class="star-point" src="images/bi_star-fill.png">
-          </div>
-          <div class="navigation-country-switzerland">
-            <img src="images/ep_location.png">
-            <p class="location-country-text-switzerland">Brazil</p>
+        <h5 class="trendy-text">Trendy</h5>
+        <h1 class="out-trending-packages-text">
+          Our Trending Tour <br />Packages
+        </h1>
+        <div class="trending-your-packages-items">
+          <div class="switzerland-container">
+            <img class="photo-switzerland" src="images/hometour-packages-switzerland.png"/>
+            <img class="img-swiss" src="images/swiss-flag.png" />

```

Comment:
burda ki 2 img bir div'in içinde olsun o div relative olsun ve ona göre pozisyonlansın
büyük `image width:100%;` olsun